README.md

```diff
@@ -151,3 +151,31 @@ To test the NGINX Kubernetes Gateway run:
 ```
 make unit-test
 ```
+
+# Release Process for NGINX Kubernetes Gateway
+
+NGINX Kubernetes Gateway uses semantic versioning for its releases. For more information see https://semver.org.
+
+Warning: Major version zero (0.y.z) is reserved for development, anything MAY change at any time. The public API is not stable.
+
+### Steps to create a release.
+
+1. Create a release branch from main, use the naming format: release-MAJOR.MINOR.
+
+2.  Create a release candidate tag, use the naming format: vMAJOR.MINOR.PATCH-rc.N (N must start from 1 and monotonically increase with each release candidate).
```

Comment:
Does the additional space render in the HTML? Best to remove it to make sure it's aligned correctly.

Reply:
Good catch.  I deleted it.  Thanks!